lib/generators/solidus/auth/install/templates/config/initializers/devise.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
-Devise.secret_key = SecureRandom.hex(50).inspect
+Devise.secret_key = '<%= SecureRandom.hex(64) %>'
```

Comment:
Can you explain how this is different? In your commit message:

> This is the default configuration Devise uses. What we had previously sets the `Devise.secret_key` to a random value at application boot that is not known across different instances of the same application.

What would make this new configuration work across many instances of the same application?

Reply:
Oh I see now. This generates a value in the application. The commit message doesn't mention that this is a template used by a template used by a generator, to generate an initializer in an application, but if I had been more attentive I would have picked up on this I guess.

Reply:
Is there a reason we cannot rename this file `devise.rb.erb`?